vms/example/xsvm/warp.go

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package xsvm
+
+import (
+	"context"
+
+	"github.com/ava-labs/avalanchego/snow/engine/common"
+	"github.com/ava-labs/avalanchego/vms/platformvm/warp"
+)
+
+// acp118Verifier allows signing all warp messages
+type acp118Verifier struct{}
```

Comment:
Is it worth adding a compile-time check that this struct implements the necessary interface?

Reply:
sure